test/Garnet.test/RespSetTest.cs

```diff
@@ -273,8 +273,86 @@ public void CanDoSScanWithCursor()
             Assert.AreEqual(l, $"value:{setEntries.Length - 1}");
         }
 
+        [Test]
+        public void CanDoSdiff()
+        {
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var db = redis.GetDatabase(0);
+
+            var key1 = "key1";
+            var key1Value = new RedisValue[] { "a", "b", "c", "d" };
+
+            var key2 = "kye2";
+            var key2Value = new RedisValue[] { "c" };
+
+            _ = db.SetAdd(key1, key1Value);
+            _ = db.SetAdd(key2, key2Value);
+
+            var result = (RedisResult[])db.Execute("SDIFF", key1, key2);
+            Assert.AreEqual(3, result.Length);
+
+            Assert.IsTrue(Array.Exists(result, t => t.ToString().Equals("a")));
+            Assert.IsTrue(Array.Exists(result, t => t.ToString().Equals("b")));
+            Assert.IsTrue(Array.Exists(result, t => t.ToString().Equals("d")));
```

Comment:
Might be a bit cleaner:

var strResult = result.Select(r => r.ToString()).ToArray();
var expectedResult = new[] { "a", "b", "d" };
Assert.IsTrue(strResult.OrderBy(t => t).SequenceEqual(expectedResult.OrderBy(t => t)));